nucleus/payara-modules/notification-core/src/main/java/fish/payara/nucleus/notification/admin/BaseNotificationConfigurer.java

```diff
@@ -123,12 +126,33 @@ public Object run(final NotificationServiceConfiguration notificationServiceConf
                         applyValues(c);
                         notificationServiceConfigurationProxy.getNotifierConfigurationList().add(c);
                         actionReport.setActionExitCode(ActionReport.ExitCode.SUCCESS);
+
+                        ParameterMap params = new ParameterMap();
+                        params.add("enabled", Boolean.FALSE.toString());
```

Comment:
Can this be set to the value of the _enabled_ parameter please? One of the goals of this is to skip out the step of having to enable the notifiers for the healthcheck and request tracing services.

Can you also add in the _dynamic_ parameter as well please?